Dockerfile

```diff
@@ -1,23 +1,25 @@
+FROM nikolaik/python-nodejs:python3.7-nodejs10
```

Comment:
FYI the main motive was to allow easier CI integration, now it's optional but I guess it's still a safer practice to have only one base